### ISSUE TITLE
Baidu Netdisk/PKU Netdisk Timeout

### ISSUE THREAD

**Issue by nmeegan** [ChemJeff/SOLD-Net#1]
Unable to download pre-trained models, train set and test set for synthetic data from Baidu Netdisk or PKU Netdisk. Page times out.

**Comment by ChemJeff** [comment]
Is something wrong with your network config? I've just checked the links and they all work well now.

**Comment by nmeegan** [comment]
Hmm, it's likely an issue with my network then. Will try another configuration as soon as I can. Thank you for the swift response.

**Comment by nmeegan** [comment]
Hello. The issue was with my network configuration, the links are working now. Apologies for any inconveniences.
